docs/RestApis.md

```diff
@@ -306,7 +306,7 @@ http://<host>:<port>/commands/database/namespaces
 
 
 ## Describe metric
-`Describe metric` command retrieves the schema descriptor of a specific metric.
+`Describe metric` command retrieves the schema descriptor and custom shard interval and retention (if present) of a specific metric.
```

Comment:
I think it might be a good idea to link out `interval` and `retention` to the related docs section because the user might fall at this point w/o owning anything about these topics.

Reply:
Thanks, it's a good idea. I'll add it (after I found the way to do it )